### PR TITLE
Display name-id in get data dialog ComboBox

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: src/model/statistics/EntityItem.cpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            return this->nameId();
+        default:
+            return QVariant();
+    }
+}
+
+QString EntityItem::nameId() const
+{
+    QString nameId = name();
+    nameId.append("-");
```
Comment (jparisu) [inline src/model/statistics/EntityItem.cpp:43]: I would suggest a way of merge name and string with a char or format that a simple name could not have, for example `name:<id>`

**Review by jparisu — APPROVED** [review]
LGTM

NIT comment to discuss

File: src/model/statistics/EntityItem.cpp (hunk 1 above)
Comment (rsanchez15) [reply src/model/statistics/EntityItem.cpp:43]: Do you mean `<name>:<id>` or `name:<id>`?

Comment (jparisu) [reply src/model/statistics/EntityItem.cpp:43]: Second one.
`<name>:'<'<id>'>'`

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**